app/views/templates/rulingCoverLetterTemplate.scala.html

```diff
@@ -169,10 +169,10 @@ <h4 class="govuk-heading-s">@Html(messages("pdf.ruling.section.askForReview.guid
                 @messages("pdf.ruling.section.askForReview.solsTeam.cannotWrite")
             </p>
             <p class="govuk-body-s">
-                @Html(messages("pdf.ruling.section.askForReview.solsTeam.emailText", messages("pdf.ruling.section.askForReview.solsTeam.email"), messages("pdf.ruling.section.askForReview.solsTeam.email")))
+                @Html(messages("pdf.ruling.section.askForReview.solsTeam.emailText"))
```

Comment:
This should be `pdf.ruling.section.askForReview.solsTeam.emailText` not `pdf.ruling.section.askForReview.solsTeam.emailText`. `pdf.ruling.section.askForReview.solsTeam.emailText` has been removed so it is not rendering.